backend/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+export PYTHONUNBUFFERED=TRUE
+mkdir -p /logs
+
+python image_similarity/main.py 2>&1 | tee /logs/gunicorn_image_similarity.log &
+python manage.py showmigrations | tee /logs/show_migrate.log
+python manage.py migrate | tee /logs/command_migrate.log
+python manage.py showmigrations | tee /logs/show_migrate.log
+python manage.py build_similarity_index 2>&1 | tee /logs/command_build_similarity_index.log
+
+python manage.py createadmin -u $ADMIN_USERNAME $ADMIN_EMAIL 2>&1 | tee /logs/command_createadmin.log
+
+echo "Running backend server..."
+
+python manage.py rqworker default 2>&1 | tee /logs/rqworker.log &
+
+if [ "$DEBUG" = 1 ]
+then
+    echo "develompent backend starting"
+    gunicorn --worker-class=gevent --timeout 36000 --reload --bind backend:8001 --log-level=info ownphotos.wsgi 2>&1 | tee /logs/gunicorn_django.log
+else
+    echo "production backend starting"
+    gunicorn --worker-class=gevent --timeout 3600 --bind backend:8001 --log-level=warning ownphotos.wsgi 2>&1
```

Comment:
This makes sense, if you have sensible logging and currently we don't. We should also switch this back to info, because a lot of errors are apparently only logged as info.

Reply:
For me no problem, the important bit here is the --reload that allow code hot reload

Reply:
Re-added the tee and info logging in production. It's not good practice though. Please open a ticket to keep track of this and change it sooner or later.

Reply:
BTW tee should not be necessary cause all logs are also visible through docker.
For example callnig:

`docker-compose logs backend -t 100 -f`

will show last 100 backend logs line and will keep listening as tail -f does on files.

Reply:
Alright. I added LibrePhotos/librephotos/issues/170 . I still have issues seeing exceptions as stated in the other pull request. I don't know the reason for that yet.

Reply:
What you don't see? a little more detail would help